docs/en/Configuring-Alluxio-with-secure-HDFS.md

```diff
@@ -152,6 +153,12 @@ Next, you can run a simple example program:
 $ bin/alluxio runTests
 ```
 
+For this test to succeed, you need to make sure that the login user of Alluxio cli, i.e. `bin/alluxio` has
+read/write access to the HDFS directory mounted to Alluxio. If not specified in `./conf/alluxio-site.properties`,
```

Comment:
I think the structure would more clear if you say something like:

By default, the login user is ... To change the login user, set ...

Reply:
Done.